zebra-chain/src/sprout/tests/tree.rs

```diff
@@ -101,7 +101,7 @@ fn incremental_roots_with_blocks_for_network(network: Network) -> Result<()> {
             &*vectors::MAINNET_FINAL_SPROUT_ROOTS,
             MAINNET_FIRST_JOINSPLIT_HEIGHT,
         ),
-        Network::Testnet => (
+        Network::Testnet(_) => (
             &*vectors::TESTNET_BLOCKS,
             &*vectors::TESTNET_FINAL_SPROUT_ROOTS,
             TESTNET_FIRST_JOINSPLIT_HEIGHT,
```

Comment:
It seems like this code has been copied around a lot, maybe we should replace it with methods in `test::vectors` that take `is_mainnet: bool`?

This can happen in an earlier PR, and then all this test code doesn't need to change at all in this PR.